tools/azure-pipelines/jobs-template.yml

```diff
@@ -161,6 +161,6 @@ jobs:
     # TODO remove pre-commit tests script by adding the tests to the nightly script
 #    - script: FLINK_DIR=build-target ./flink-end-to-end-tests/run-pre-commit-tests.sh
 #      displayName: Test - precommit 
-    - script: FLINK_DIR=`pwd`/build-target flink-end-to-end-tests/run-nightly-tests.sh
+    - script: ${{parameters.environment}} FLINK_DIR=`pwd`/build-target flink-end-to-end-tests/run-nightly-tests.sh
```

Comment:
should be a separate commit, and possibly even JIRA.

Reply:
Why?

What this does is passing the `PROFILE` variable to the script. This is necessary for deciding whether we are in a jdk11 setting or not. ... I see a clear relationship to the title of the JIRA I'm working on.

Reply:
because the script was _already_ relying on this being set for the avro schema registry. This fixes a bug, it's not an improvement.